examples/hello_world.rs

```diff
@@ -64,6 +64,8 @@ fn main() {
         "Markdown viewer",
         eframe::NativeOptions::default(),
         Box::new(move |cc| {
+            egui_extras::loaders::install(&cc.egui_ctx);
```

Comment:
I have been keeping an eye on the new image api. Exciting! 

Is it not possible to allow libraries to install the loaders if they are needed? I would like to have `egui_extras` as an implementation detail. And more importantly `egui_commonmark` loads images by default and it won't be able to do that anymore if it can't setup the loaders itself!

In short: Is it possible to make this "Immediate mode safe" or add a way to check if the loaders have not been installed so that `egui_commonmark` can do it?

Reply:
As a baseline, the error message caused by not having any loaders should be improved. This stuff should also be very prominent in any documentation. As for making this immediate-mode safe, I think if we add a check to see if a `Context` already contains the `egui_extras` loaders, then we could make calling `egui_extras::loaders::install` multiple times on the same `Context` safe, so libraries could do it.